File: Assignment 1/medcare-appointment-manager/src/main/java/org/example/medcareappointmentmanager/business/dto/AppointmentDTO.java
package org.example.medcareappointmentmanager.business.dto;

import org.example.medcareappointmentmanager.data.AppointmentStatus;

import java.sql.Time;
import java.util.Date;

public record AppointmentDTO(Long id, String patientName, DoctorDTO doctor, MedicalServiceDTO medicalService, Date data, Time time, AppointmentStatus status) {
}

File: Assignment 2/medcare-appointment-manager-frontend/src/types/AppointmentDTO.ts
import { DoctorDTO } from "./DoctorDTO";
import { MedicalServiceDTO } from "./MedicalServiceDTO";

export interface AppointmentDTO {
  id: number | null;
  patientName: string;
  doctor: DoctorDTO;
  medicalService: MedicalServiceDTO;
  date: string;
  time: string;
  status: string;
}